Lesson33/Selcan/lesson33-app/src/Exercise5/index.jsx

```diff
@@ -0,0 +1,36 @@
+// Exercise 5: Mouse Coordinates
+// Build a component that displays the current X and Y coordinates of the mouse when a button is clicked.
+    
+import { useState, useEffect } from "react";
+
+export const Exercise5 = () => {
```

Comment:
The solution works however it's a bit too complex. For this exercise, it would be sufficient to add a click handler function that updates `coords` state with `event.clientX` and `event.clientY` when the button is clicked. So no need to keep track of `displayCoords` state nor use `useEffect`